src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  if (days < 3) {
+    rentalCost = days * 40;
+  } else if (days < 7) {
+    rentalCost = days * 40 - 20;
+  } else {
+    rentalCost = days * 40 - 50;
```

Comment:
You can simplify the function by returning directly within the `if` blocks and avoid using `else` or `else if` after a return statement. This will make the code cleaner and more readable.